src/Synthesizer/Modifiers/Envelopes.hs

```diff
@@ -85,6 +85,7 @@ getSustainSteps sustainLevel eventLength attackLength decayLength samplingRate =
   where
     sustainLength = eventLength - attackLength - decayLength
 
+-- | Calculate the sustain steps based on the release length and the sustain level
```

Comment:
I think this calculates the release steps not the sustain steps

Reply:
Fixed in fcc0c66